adhocracy4/ratings/templatetags/react_ratings.py

```diff
@@ -45,6 +50,7 @@ def react_ratings(context, obj):
         'userRatingId': user_rating_id,
         'isReadOnly': not has_rate_permission,
         'style': 'ideas',
+        'would_have_perm': would_have_rate_permission,
```

Comment:
while the would_have_perm is set it is not used in the react_ratings.jsx. The rating buttons are disabled if isReadOnly is set to true. either the button would have to be enabled if `!isReadOnly || would_have_perm` or the isReadOnly could be adapted in the manner suggested above.

for login redirections maybe the lines [58](https://github.com/liqd/adhocracy4/blob/588e61dccf739f6836458a747fc4dfc73b6432bb/adhocracy4/ratings/static/ratings/react_ratings.jsx#L58) and [79](https://github.com/liqd/adhocracy4/blob/588e61dccf739f6836458a747fc4dfc73b6432bb/adhocracy4/ratings/static/ratings/react_ratings.jsx#L79) of react_ratings.jsx would probably have to be adapted.

Reply:
I wasn't quite sure, if this didn't have to wait for the next user story, because there should be a pop-up to login for rating. But with the isReadOnly the user is simply forwarded to login, and the pop-up has to wait until the next user story.